exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -591,6 +591,119 @@ func TestExporter_createMeasure(t *testing.T) {
 	}
 }
 
+func TestExporter_makeReq_withCustomMonitoredResource(t *testing.T) {
+	m, err := stats.NewMeasureFloat64("test-measure", "measure desc", "unit")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer stats.DeleteMeasure(m)
+
+	key, err := tag.NewKey("test_key")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cumView, err := stats.NewView("cumview", "desc", []tag.Key{key}, m, stats.CountAggregation{}, stats.Cumulative{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := stats.RegisterView(cumView); err != nil {
```

Comment:
Could we make this consistent with cumView.Unregister below by using
```go
cumView.Subscribe()
```
and then below:
```go
defer cumView.Unsubscribe()
```

I'll actually file a bug to either add stats.UnregisterView or to remove stats.RegisterView entirely.

Reply:
Sure, updated. I didn't update other tests though, because if I do so I'll get an error saying "cannot register measure "test-measure"; another measure with the same name is already registered".